src/components/CIPipelineN/CIPipeline.tsx

```diff
@@ -536,38 +531,17 @@ export default function CIPipeline({
         close()
     }
 
-    const onDelete = async () => {
-        await deleteCIPipeline(
-            formData,
-            ciPipeline,
-            formData.materials,
-            Number(appId),
-            Number(workflowId),
-            false,
-            formData.webhookConditionList,
-        )
+    const onClose = () => {
         setPageState(ViewType.FORM)
         handleClose()
-        deleteWorkflow(appId, Number(workflowId))
     }
 
-    const closeCIDeleteModal = (): void => {
-        setShowDeleteModal(false)
+    const onDelete = () => {
+        onClose()
+        setPageState(ViewType.FORM)
```

Comment:
OnClose already have these 2 lines. Instead of creating a new function should use `onClose`